docker-compose.yml

```
@@ -1,7 +1,7 @@
 version: '2'
 services:
   platform-docs:
-    command: sh -c "npm run dev"
+    command: sh -c "npm ci && npm run dev:compose"
```

No need to run `ci` it's part of the `dev:compose` script.
```suggestion
    command: sh -c "npm run dev:compose"
```

---

It needs the CI there. It can't be in the script part in package.json. I have removed it from the dev:compose